packages/client/src/views/OfficeHome/inProgress/inProgress.test.tsx

```diff
@@ -902,15 +904,17 @@ describe('In Progress tab', () => {
       await new Promise((resolve) => {
         setTimeout(resolve, 100)
         testComponent.update()
-        const data = testComponent.find(Workqueue).prop('content')
+        const data = testComponent
+          .find(Workqueue)
+          .prop<Array<Record<COLUMNS, string>>>('content')
         const EXPECTED_DATE_OF_REJECTION = formattedDuration(
           new Date(birthNotificationSentDateStr)
         )
 
-        expect(data[0].id).toBe('f0a1ca2c-6a14-4b9e-a627-c3e2e110587e')
-        expect(data[0].name).toBe('anik hoque')
-        expect(data[0].notificationSent).toBe(EXPECTED_DATE_OF_REJECTION)
-        expect(data[0].event).toBe('Birth')
+        expect(data[1].id).toBe('f0a1ca2c-6a14-4b9e-a627-c3e2e110587e')
+        expect(data[1].name).toBe('anik hoque')
+        expect(data[1].notificationSent).toBe(EXPECTED_DATE_OF_REJECTION)
+        expect(data[1].event).toBe('Birth')
```

Comment:
@Zangetsu101 wonder why the Birth is now `[1]` of the array and not `[0]` like before. Like the rendering order would have changed for some reason.... 🤔

Reply:
Fixed in commit [97efb36](https://github.com/opencrvs/opencrvs-core/pull/4587/commits/97efb366da85a428526d032e7b38d23b47590cca)